openapi.yaml

```diff
@@ -7002,6 +7056,7 @@ components:
         * `release-age` - average timedelta between releases of the same alignment (as determined by the commit DAG). Quantiles are applied.
         * `release-tag-age` - average timedelta between releases by tag of the same alignment (as determined by the commit DAG). Quantiles are applied.
         * `release-branch-age` - average timedelta between releases by branch of the same alignment (as determined by the commit DAG). Quantiles are applied.
+        * `release-time-to-deploy` - average time passed since release happened until deployed.
```

Comment:
I understood that PRs in repos without deployments are excluded here, but is this metric computed only on PRs deployed? or is a PR released in a repo but not yet deployed also contributing to this metric?

Reply:
Sorry for the confusion, this adds the metric to `/metrics/releases`. I know it's out of scope but it was hard to keep it in my head.

Replacing "PR" with "release" and answering the original question: computed only on releases deployed.

Leaving "PR" and considering `/metrics/prs`: also computed only on deployed PRs.

Reply:
Ok, that's good then!